app/models/request/long_read.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+require_dependency 'request'
+class Request
+  # LongRead request
+  class LongRead < CustomerRequest
+    self.sequencing = true
+
+    has_metadata as: Request do
+      custom_attribute(:library_type, required: true, validator: true, selection: true)
+    end
+
+    validates :state, presence: true
+
+    destroy_aasm
```

Comment:
It might be unnecessary to do this in the end, but we can make that decision later.